src/api/Feed.js

```diff
@@ -221,7 +222,7 @@ class Feed extends Base {
         const cache = this.getCache();
         const cacheKey = this.getCacheKey(id);
         cache.set(cacheKey, {
-            hasError: !!this.hasError,
```

Comment:
Does any other module use this value from the cache?

Reply:
No, only `items` value fro the cache is used in other module.